Fastfile.private

```diff
@@ -138,12 +137,15 @@ reflected in `Info.plist` during building."
     end
   end
 
-  desc "Executes the tests for the project using `scan`. This lane uses the configuration mapped to `:test`."
-  private_lane :run_tests do
+  desc "Executes the tests for the project using `scan`."
+  private_lane :run_tests do |options|
+
+    build_configuration_key = options[:build_configuration]
+    build_configuration = get_build_configuration(build_configuration: build_configuration)
```

Comment:
`build_configuration = get_build_configuration(build_configuration: build_configuration_KEY)` key missing

Reply:
Yes, I'll add it 👍